redbox-core/redbox/chains/parser.py

```diff
@@ -58,6 +58,7 @@ def _transform(self, input: Iterator[Union[str, BaseMessage]]) -> Iterator[Any]:
                         dispatch_custom_event(RedboxEventType.response_tokens, data=new_tokens)
                         field_length_at_last_run = len(field_content)
                         yield self.pydantic_schema_object.model_validate(parsed)
+        yield self.pydantic_schema_object.model_validate(parsed)
```

Comment:
what happens when `parsed` is undefined?

Reply:
ah good catch